### PR TITLE
Fixes part of #4177: Dark mode Topic Activity Part 3 (Revision Card Tab)

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fixes part of #4177: Dark mode Topic Activity Part 3 (Revision Card Tab)

Dark Mode Mockup:-
https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/1adacef0-f245-4115-a0e0-d1c2e7d5a0bb/specs/

|Default|Dark Mode|
|:----------------------------------------:|:-----------------------------------------:|
|  | Screenshot_2022-05-31-12-23-27-332_com android camera |

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/res/values-night/color_palette.xml
Comment (rt4914) [inline app/src/main/res/values-night/color_palette.xml]: This change should be removed once your part-2 is merged

**Review by rt4914 — CHANGES_REQUESTED** [review]
This PR contains some unwanted changes in all the files.

Either remove those changes or wait for part-2 to get merged and then merge with develop and update this PR

File: app/src/main/res/values-night/color_palette.xml
Comment (MohitGupta121) [reply app/src/main/res/values-night/color_palette.xml]: > This change should be removed once your part-2 is merged

okay sure

**Comment by MohitGupta121** [comment]
> This PR contains some unwanted changes in all the files.
> 
> Either remove those changes or wait for part-2 to get merged and then merge with develop and update this PR

Okay understand.

File: app/src/main/res/values-night/color_palette.xml
Comment (MohitGupta121) [reply app/src/main/res/values-night/color_palette.xml]: > This change should be removed once your part-2 is merged

Now Part 2 is merge, I take pull from the develop branch but if I remove this value then dark mode this component is not work. As after pull this not add new value that's why?

https://github.com/oppia/oppia-android/blob/f99fa1d8653e23f8d22e1584a9fb1793aaa19acd/app/src/main/res/values/component_colors.xml#L35

This component using that values.

Comment (rt4914) [reply app/src/main/res/values-night/color_palette.xml]: Just make sure that this value is getting used somwhere.

Earlier when I reviewed it. the code was written in such a way that it looked as if it was not getting used anywhere.

If it is getting used anywhere, don't remove it.

File: app/src/main/res/values/color_defs.xml
Comment (rt4914) [inline app/src/main/res/values/color_defs.xml]: `Bbackground` word cannot be used in `color_defs` file. 
First check if similar color already exists, if not, then name it `color_def_oppia_white_2` or something of similar pattern.

File: app/src/main/res/values/component_colors.xml
Comment (rt4914) [inline app/src/main/res/values/component_colors.xml]: `Activity ` -> `Fragment`

File: app/src/main/res/values/component_colors.xml
Comment (rt4914) [inline app/src/main/res/values/component_colors.xml]: `component_color_topic_revision_fragment_background_color `

**Review by rt4914 — CHANGES_REQUESTED** [review]
@MohitGupta121 PTAL

File: app/src/main/res/values/color_defs.xml
Comment (MohitGupta121) [reply app/src/main/res/values/color_defs.xml]: > 

Okay I'll do that.

File: app/src/main/res/values/component_colors.xml
Comment (MohitGupta121) [reply app/src/main/res/values/component_colors.xml]: > `Activity ` -> `Fragment`

Okay

File: app/src/main/res/values/component_colors.xml
Comment (MohitGupta121) [reply app/src/main/res/values/component_colors.xml]: > `component_color_topic_revision_fragment_background_color `

Okay sure I'll do that.

File: app/src/main/res/values-night/color_palette.xml
Comment (MohitGupta121) [reply app/src/main/res/values-night/color_palette.xml]: > Just make sure that this value is getting used somwhere.
> 
> Earlier when I reviewed it. the code was written in such a way that it looked as if it was not getting used anywhere.
> 
> If it is getting used anywhere, don't remove it.

I compare it develop and it from develop and it's used in layout.

**Comment by MohitGupta121** [comment]
@rt4914 I commit requested changes, can you please review it again.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Comment by MohitGupta121** [comment]
> LGTM, Thanks.

Thanks you

**Pull request merged**
